packages/web-config-server/src/apiV1/dataBuilders/DataBuilder.js

```diff
@@ -80,6 +80,15 @@ export class DataBuilder {
       eventId,
       ...additionalQueryConfig,
     });
+    if (
+      additionalQueryConfig.entityAnswerIdToName &&
+      additionalQueryConfig.entityAnswerIdToName.dataElementCodes
```

Comment:
We now support [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) in our codebase:

```suggestion
      additionalQueryConfig?.entityAnswerIdToName?.dataElementCodes
```

Reply:
Thanks Kostas, love this shorter expression!